Use cogs:Jobs and Tasks and integrate with job-controller

benjay10 opened the pull request:
This PR makes this service integrate with the job-controller-service. It listens for scheduled tasks with operation "validate" and starts the process from the data in the inputContainer of the task. Some function have been removed or rewritten because of this. When the task is finished, its status is set to "success" and the resultsContainer will point to the inputContainer, because no file has been created or modified, only triples in the database. Ideally, we should have a resultsContainer that can point to individuals in the data as well.

Some constants have also been moved to a separate file and some imports are now done differently.

benjay10 committed:
Integration with job-controller

Using tasks defined by the model of the job-controller. Some functions
and variables have been moved.

benjay10 committed:
Updated README about Task model

benjay10 committed:
Removed debugger statement

cecemel commented on env.js:
```diff
@@ -0,0 +1,53 @@
+  asj:          'http://data.lblod.info/id/automatic-submission-job/',
+};
+
+export const PREFIXES = (() => {
```
same comment as automatic submission (let's keep the discussion there)

benjay10 committed:
Some files now using file-service structure

Use queries for that data structure in the correct places.

benjay10 committed:
Work better with logical files

Just like in some other services in the flow, we are now using logical
files and include those in the resultsContainer for the task.

benjay10 committed:
Rollback env.getPrefixes() to env.PREFIXES

cecemel committed:
Take into account the task:Task when checking if

remote data object has been saved
